tests/readme.md

```diff
@@ -2,9 +2,8 @@ This directory contains the test cases.
 
 Each test must have a filename of the form: ``t*.nim``
 
-**Note:** Tests are only compiled by default. In order to get the tester to
-execute the compiled binary, you need to specify a spec with an ``action`` key
-(see below for details).
+**Note:** Testament is only aware of tests under a directory (eg `tests/foo/`) and will ignore
+top-level tests like `tests/tbar.nim`.
```

Comment:
Sounds like a bug to me. We should fix this given that testament is now a part of the official Nim distribution.

Reply:
I agree; tracking this here https://github.com/nim-lang/Nim/issues/14378
this should be fixed in a different PR though, I'm just reflecting current reality here.